pkgs/applications/networking/browsers/tor-browser-bundle-bin/default.nix

```diff
@@ -409,9 +409,7 @@ stdenv.mkDerivation rec {
     LD_LIBRARY_PATH=$libPath $TBB_IN_STORE/TorBrowser/Tor/tor --version >/dev/null
 
     echo "Checking tor-browser wrapper ..."
-    DISPLAY="" MOZ_ENABLE_WAYLAND="" WAYLAND_DISPLAY="" XAUTHORITY="" \
-      XCURSOR_PATH="" XDG_RUNTIME_DIR="" XDG_SESSION_TYPE="" \
-      DBUS_SESSION_BUS_ADDRESS="" TBB_HOME=$(mktemp -d) \
+      TBB_HOME=$(mktemp -d) \
```

Comment:
So while this does indeed work on my system, I am wondering why DISPLAY, XAUTHORITY and DBUS_SESSION_BUS_ADDRESS were unset here prior to https://github.com/NixOS/nixpkgs/pull/168497.
Maybe this used to be there for platforms where builds are not sandboxed?

Reply:
yeah maybe

sandbox builds were enabled on linux by default in [2019](https://nixos.org/manual/nix/stable/release-notes/rl-2.2.html?highlight=sandbox#release-22-2019-01-11) while those were added in [2017](https://github.com/nixos/nixpkgs/commit/ecd0e1a2c736a9a609cd820de0cb2861129482c3)

Reply:
But other platforms (mac?) don't have sandboxing yet, so we should probably not touch it yet?

Reply:
this package only supports x86_64-linux and i686-linux

Reply:
I think there are valid use cases for disabling the build sandbox on linux, at least there used to be, e.g. if you are stuck with a kernel you have no control over, that does not allow for user namespace. But maybe its ok to rely on the sandbox these days.. not sure

Reply:
https://github.com/NixOS/nix/issues/2651 seems to be such an edge case

Reply:
its not a big enough concern for us 

im quite sure that in nixpkgs we assume that sandboxing is on

Reply:
If you are sure, then lets go ahead